src/pe/import.rs

```diff
@@ -293,6 +298,32 @@ impl<'a> SyntheticImportDirectoryEntry<'a> {
             import_address_table,
         })
     }
+
+    pub fn get_imports_rva<T: Bitfield<'a>>(
+        &self,
+    ) -> error::Result<Vec<SyntheticImportAddressTableEntry<'a>>> {
+        let mut out = Vec::<SyntheticImportAddressTableEntry>::new();
+        if let Some(import_lookup_table) = &(self.import_lookup_table) {
+            for (i, lookup_entry) in import_lookup_table.iter().enumerate() {
+                match lookup_entry {
+                    SyntheticImportLookupTableEntry::OrdinalNumber(_) => {
+                        continue;
+                    }
+                    SyntheticImportLookupTableEntry::HintNameTableRVA((_, v)) => {
+                        let import_rva = (self.import_directory_entry.import_address_table_rva
+                            as u64)
+                            + ((i * std::mem::size_of::<T>()) as u64);
```

Comment:
You mentioned needing to know the size of the PE; this information is usually in the parsing context but I’m not sure if you have access to it here, haven’t looked at this code in a while. Or is the mem::size_of sufficient for this ?

Reply:
`mem::size_of` is sufficient for this, but honestly passing a boolean called `is_64bit`  would be a cleaner option IMO.